array_sum.py

```diff
@@ -0,0 +1,35 @@
+Method 1:
+# Python 3 code to find sum of elements in given array 
+def _sum(arr,n): 
```

Comment:
Why did you use _ as perfix

Reply:
_ is used to return a value in an array or function